http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ChannelPipelineInitializer.java

```diff
@@ -75,15 +77,16 @@ public void channelCreated(Channel ch) {
         }
 
         pipeline.addLast(new FutureCancelHandler());
+        pipeline.addLast(new LoggingHandler(LogLevel.DEBUG));
```

Comment:
This will only log debug logs if you have debug logging enabled for "io.netty".

We could expose the log level to the customer but I'm not exactly sure why we'd need to. Since you can already control logging at a granular level, you can just set io.netty to debug even if you want it all the time.

Reply:
Make sense.